source/whats-new.txt

```diff
@@ -12,12 +12,26 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 2.22 <version-2.22>`
 * :ref:`Version 2.21 <version-2.21>`
 * :ref:`Version 2.20 <version-2.20>`
 * :ref:`Version 2.19 <version-2.19>`
 
 .. _upcoming-breaking-changes:
 
+.. _version-2.22:
+
+What's New in 2.22
+------------------
+
+The 2.22 driver release includes the following new features:
+
+- Improvements and bug fixes to the LINQ3 integration.
+- Added support for ``$set`` and ``$vectorSearch`` aggregation stages.
+- Added synonym support for Atlas Search.
```

Comment:
Question:
Are there any docs that can be linked to that describe these features?